### DISCUSSION TITLE
Being able to customize or add new types of "Block" in the rich text editor

### DISCUSSION THREAD

**Discussion by felixfong227** [saleor/saleor#16040] [Q&A]
Hello, right now, I am faced with an issue of not being able to insert an image into my product description.

The current implantation only consists of "textual" data
![image](https://github.com/saleor/saleor/assets/13918481/e6953a2a-41bb-43a0-87dc-97e5fd8f4a0f)

It would be great to have images and videos as well. Additionally, it would be beneficial if Saleor provided an SDK for developers to build their own custom experiences.

Thank you

**Comment by timuric** [comment]
Hey @felixfong227 have you considered using attributes to model your product description instead of default WYSIWYG description form?

**Reply by bnkwines** [reply]
i've patched the admin project with more editorjs plugins, then you've to handle the new data on your frontends
